### PR TITLE
chore: `Skip` param added to `Utils::map_input()`

### PR DESCRIPTION
**Pull request by kidunot89**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Adds a `skip` param to the `Utils::map_input()` function.

Does this close any currently open issues?
------------------------------------------

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** …

**WordPress Version:** …

### CODE REVIEW

File: src/Utils/Utils.php [hunk 1]
```diff
@@ -34,15 +34,16 @@ public static function get_query_id( string $query ) {
 	}
 
 	/**
-	 * Maps new input query args and sa nitizes the input
+	 * Maps new input query args and sanitizes the input
 	 *
 	 * @param mixed|array|string $args The raw query args from the GraphQL query
 	 * @param mixed|array|string $map  The mapping of where each of the args should go
+	 * @param array              $skip Fields to skipped and not be added to the output array.
```
Comment (justlevine) [inline src/Utils/Utils.php:41]:
```suggestion
	 * @param string[]            $skip Fields to skipped and not be added to the output array.
```

File: src/Utils/Utils.php [hunk 2]
```diff
@@ -51,6 +52,9 @@ public static function map_input( $args, $map ) {
 		$query_args = [];
 
 		foreach ( $args as $arg => $value ) {
+			if ( in_array( $arg, $skip, true ) ) {
```
Comment (justlevine) [inline src/Utils/Utils.php:55]:
```suggestion
			if ( [] !== $skip && in_array( $arg, $skip, true ) ) {
```

**Comment by justlevine** [comment]
Left some tiny suggestions, but otherwise I love it.

File: src/Utils/Utils.php (hunk 1 above)
Comment (kidunot89) [reply src/Utils/Utils.php]: @justlevine Done

File: src/Utils/Utils.php (hunk 2 above)
Comment (kidunot89) [reply src/Utils/Utils.php]: @justlevine And done

**Review by jasonbahl — APPROVED** [review]

**Pull request merged**